common/presenters/allocations-to-table.test.js

```diff
@@ -92,30 +92,30 @@ describe('#allocationsToTable', function() {
     })
     it('returns the total moves count on the first cell', function() {
       expect(output.rowsForAllocationTable[0][0]).to.deep.equal({
-        html: 3,
+        text: 3,
         attributes: {
           scope: 'row',
         },
       })
     })
     it('returns the created date on the second cell', function() {
       expect(output.rowsForAllocationTable[0][1]).to.deep.equal({
-        html: '20 Apr 2020',
+        text: '20 Apr 2020',
       })
     })
     it('returns location_from on the third cell', function() {
       expect(output.rowsForAllocationTable[0][2]).to.deep.equal({
-        html: 'HOLME HOUSE (HMP)',
+        text: 'HOLME HOUSE (HMP)',
       })
     })
     it('returns location_to on the fourth cell', function() {
       expect(output.rowsForAllocationTable[0][3]).to.deep.equal({
-        html: 'HMP Ashfield',
+        text: 'HMP Ashfield',
       })
     })
     it('returns move date on the fifth cell', function() {
       expect(output.rowsForAllocationTable[0][4]).to.deep.equal({
-        html: '3 May 2020',
+        text: '3 May 2020',
       })
     })
     xit('returns progress on the sixth cell', function() {
```

Comment:
Should this still be skipped?

Reply:
Not sure. It was an existing test that @barbaracassani I think added as not sure what is going to go in that column for allocations.